api/v1/Game.php

```diff
@@ -248,26 +249,23 @@ public function SetupCountries(array $configData): void
     // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
     public function SetupGameTime(array $data): void
     {
-        $_POST['user'] = 1; // should this go at some point?
         $this->SetStartDate($data['start']);
 
-        //$_POST['months'] = $data['era_planning_months']; // this should definitely go at some point
         $this->Planning($data['era_planning_months']);
 
-        //$_POST['realtime'] = $data['era_planning_realtime'];
         $this->Realtime($data['era_planning_realtime']);
-
         $str = "";
-
         $totalEras = 4;
         $str .= str_repeat($data['era_planning_realtime'] . ",", $totalEras);
-
         $str = substr($str, 0, -1);
 
         /** @noinspection SqlWithoutWhere */
         $this->getDatabase()->query(
             "UPDATE game SET game_planning_era_realtime=?, game_eratime=?",
-            array($str, $data['era_total_months'])
+            array(
+                $str,
+                max($data['era_total_months'], self::MIN_GAME_ERATIME)
```

Comment:
This is the only spot where the table column used later could even become 0. So with the max function we force a minimum value.

Reply:
alright